integration/commands_administration_test.go

```diff
@@ -159,24 +160,112 @@ func TestCommandsAdministrationListDatabases(t *testing.T) {
 	db := collection.Database()
 	name := db.Name()
 
-	actual, err := db.Client().ListDatabases(ctx, bson.D{{"name", name}})
-	require.NoError(t, err)
-	require.Len(t, actual.Databases, 1)
+	t.Run("Exists", func(t *testing.T) {
+		actual, err := db.Client().ListDatabases(ctx, bson.D{{Key: "name", Value: name}})
```

Comment:
```suggestion
		t.Parallel()

		actual, err := db.Client().ListDatabases(ctx, bson.D{{Key: "name", Value: name}})
```

Subtests should run in parallel too

Reply:
Added t.Parallel() call.